tensorflow/python/ops/math_ops.py

```diff
@@ -3204,6 +3204,16 @@ def sigmoid(x, name=None):
   """Computes sigmoid of `x` element-wise.
 
   Specifically, `y = 1 / (1 + exp(-x))`.
+  
+  The sigmoid function is a function that outputs between `[0, 1]` and is often
+  used as the logistic activation function of the last layer in a 
+  classification model. This is due to the nature of its range - for values of 
+  `x` that are positive, the function outputs a value `> 0.5` (and as `x` 
+  approaches infinity, the value approaches `1`), and for values of `x` that 
+  are negative, the function outputs a value `< 0.5` (and as `x` approaches 
+  negative infinity, the value approaches `0`). At `x = 0`, the value is `0.5`.
+  Due to this behavior, the sigmoid function is often used to map a tensor to 
+  a list of probabilities (since probabilities, as well, are between `[0, 1]`).
```

Comment:
Consider changing "...(since probabilities, as well, are between `[0, 1]`)." to "...(since probabilities are also between `[0, 1]`)."